packages/utils/telemetry-utils/src/mockLogger.ts

```diff
@@ -71,6 +71,19 @@ ${JSON.stringify(actualEvents)}`);
 expected:
 ${JSON.stringify(expectedEvents)}
 
+actual:
+${JSON.stringify(actualEvents)}`);
+            }
+    }
+
+    /** Asserts that matchAnyEvent is false for the given events, and prints the actual/expected output if not */
+    assertMatchNone(disallowedEvents: Omit<ITelemetryBaseEvent, "category">[], message?: string) {
+        const actualEvents = this.events;
+        if (this.matchAnyEvent(disallowedEvents)) {
+            throw new Error(`${message}
+disallowed events:
```

Comment:
What does this tag do?

Reply:
This is a multi-line string.  Not sure what you mean by tag.